### ISSUE TITLE
Admin Dashboard Page 2

### ISSUE THREAD

**Issue by jchen42703** [jchen42703/waiting-line-app#12]
![image](https://user-images.githubusercontent.com/34604336/154769253-fadf0a75-7124-49e2-96ed-1b3f1a45789c.png)

## Next User Button
- `POST /api/queue/pop` --> returns the front user (place 1)
  - This should cause the table to rerender 

## User Table
- Gets an array of Users from `/api/queue/all` <-- hasn't been created yet

## User Button

```
interface User {
  place: number;
  name: string;
  email: string;
  phoneNumber: string; // maybe make separate type for this
  timeJoined: number;
}
```
- States:
  - DeleteUserButton
     - toggleable

## Back Button

Basically goes to previous page in router history
